bundle.proto

```diff
@@ -81,31 +107,44 @@ message Processed {
   /// Index within the block.
   uint64 bundle_index = 3;
 }
+
 message Dropped {
   DroppedReason reason = 1;
 }
 enum DroppedReason {
+  // The bundle's blockhash expired.
   BlockhashExpired = 0;
-  // One or more transactions in the bundle landed on-chain, invalidating the bundle.
+
+  // One or more, but not all, transactions in the bundle landed on-chain, invalidating the bundle.
   PartiallyProcessed = 1;
-  // This indicates bundle was processed but not finalized. This could occur during forks.
-  NotFinalized = 2;
+
+  // Bundle was processed but not finalized. This could occur during forks.
+  // Deprecated, stop using this field.
+  NotFinalized = 2 [deprecated = true];
 }
 
 message BundleResult {
   // Bundle's Uuid.
   string bundle_id = 1;
 
   oneof result {
-    // Indicated accepted by the block-engine and forwarded to a jito-solana validator.
-    Accepted accepted = 2;
+    // Indicates accepted by the block-engine and forwarded to a jito-solana validator.
+    // Deprecated, start using `forwarded_to_validator`.
+    Accepted accepted = 2 [deprecated = true];
+
     // Rejected by the block-engine.
     Rejected rejected = 3;
+
     // Reached finalized commitment level.
     Finalized finalized = 4;
+
     // Reached a processed commitment level.
     Processed processed = 5;
-    // Was accepted and forwarded by the block-engine but never landed on-chain.
+
+    // Dropped from the mempool.
```

Comment:
whats the difference between rejected and dropped?

Reply:
suggestion:
rejected means user did something wrong (which is user-correctable)
dropped means block engine could not process it due to some internal error

Reply:
i like it. rejected = 4xx error code, dropped = 5xx error code

Reply:
you have Status for streams tho, so not exactly equal. but maybe general guideline

Reply:
maybe need to rename dropped, but dropped is representing bundles that were forwarded but did not land hence `DroppedReason` enum types. rejected bundles never got forwarded